lib/http.h

```diff
@@ -42,7 +42,7 @@ typedef enum {
 
 #ifndef CURL_DISABLE_HTTP
 
-#if defined(_WIN32) && (defined(ENABLE_QUIC) || defined(USE_NGHTTP2))
+#if defined(ENABLE_QUIC) || defined(USE_NGHTTP2) || defined(USE_MSH3)
```

Comment:
This can be simplified to just:
~~
#ifdef ENABLE_QUIC
~~~

... since it will always be defined if nghttp2 or msh3 is defined.

Reply:
building with any quic backend will define `ENABLE_QUIC` , so checking extra for `USE_MSH3` isn't necessary